lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -44,7 +44,8 @@ def related_links
       end
 
       def also_part_of_step_nav
-        step_by_step_navs = step_navs.delete_if { |step_nav| step_nav.content_id == active_step_by_step.content_id }
+        step_navs_list = step_navs_combined_list
```

Comment:
Do we need to check the combined list for the also_part of section?

Surely, if the content_item is "part_of" (not related_to) other step_navs, they will already appear in "step_navs"?

Reply:
@DilwoarH We've spoken about this in meatspace, so I'm happy for this commit to stay in.